spark_expectations/core/context.py

```diff
@@ -1564,3 +1565,23 @@ def get_stats_table_writer_config(self) -> dict:
             dict: Returns stats_table_writer_config which in dict
         """
         return self._stats_table_writer_config
+
+    def set_se_enable_error_table(self, se_enable_error_table: bool) -> None:
+        """
+
+        Args:
+            _se_enable_error_table:
+
+        Returns:
+
+        """
+        self._se_enable_error_table = bool(se_enable_error_table)
```

Comment:
as `se_enable_error_table` is already boolean, we don't need to re-cast it.

```suggestion
        self._se_enable_error_table = se_enable_error_table
```